build.gradle

```diff
@@ -617,6 +617,32 @@ task htmlValidate(type: Exec, group: 'Format') {
     ]
 }
 
+subprojects {
```

Comment:
Is this needed? Both `checker` and `framework` already list a `testannotations` sourceset.
What does this add?
If it is needed, add a comment.

Reply:
Where are you accessing it in the root project?

Reply:
at line 637 and 640 in this file.